src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -283,7 +340,7 @@ robocopy /S /NP %EXECUTION_DIR%int\%(Identity)\ %EXECUTION_DIR%native\
       <TestCommandLines Include="mklink /H %(RunnerFolderContents.RecursiveDir)%(RunnerFolderContents.Filename)%(RunnerFolderContents.Extension) %RUNTIME_PATH%\Runner\%(RunnerFolderContents.RecursiveDir)%(RunnerFolderContents.Filename)%(RunnerFolderContents.Extension)" />
 
       <!-- Copy the runtime binaries over -->
-      <RuntimePathContents Include="$(RuntimePath)\**\*" />
+      <RuntimePathContents Include="$(TestHostRootPath)\UAPLayout\*"/>
```

Comment:
Yes, I've already submitted a helix private run and I had no issues, all the tests passed.